src/EventStore.TestClient/Commands/DvuBasic/DvuBasicProcessor.cs

```diff
@@ -208,7 +208,7 @@ private void Write(Status status, int writerIdx, CommandProcessorContext context
 
             var failed = 0;
 
-            var rnd = new Random();
+            var rnd = new Random(writerIdx);
```

Comment:
The writers were getting a lot of version conflicts as random without seed would spit out the same values